aspnetcore/fundamentals/logging.md

```diff
@@ -376,72 +525,92 @@ The resulting *C:\trace.etl* file can be analyzed with PerfView as on other edit
 
 The [Microsoft.Extensions.Logging.EventLog](https://www.nuget.org/packages/Microsoft.Extensions.Logging.EventLog) provider package sends log output to the Windows Event Log.
 
+# [ASP.NET Core 1.x](#tab/aspnetcore1x)
+
 ```csharp
 loggerFactory.AddEventLog()
 ```
 
 [AddEventLog overloads](https://docs.microsoft.com/aspnet/core/api/microsoft.extensions.logging.eventloggerfactoryextensions) let you pass in `EventLogSettings` or a minimum log level.
 
+# [ASP.NET Core 2.x](#tab/aspnetcore2x)
+
+```csharp
+logging.AddEventLog()
+```
+
+---
+
 <a id="tracesource"></a>
 ### The TraceSource provider
 
 The [Microsoft.Extensions.Logging.TraceSource](https://www.nuget.org/packages/Microsoft.Extensions.Logging.TraceSource) provider package uses the [System.Diagnostics.TraceSource](https://msdn.microsoft.com/library/system.diagnostics.tracesource.aspx) libraries and providers.
 
+# [ASP.NET Core 1.x](#tab/aspnetcore1x)
+
 ```csharp
 loggerFactory.AddTraceSource(sourceSwitchName);
 ```
 
+# [ASP.NET Core 2.x](#tab/aspnetcore2x)
+
+```csharp
+logging.AddTraceSource(sourceSwitchName);
+```
+
+---
+
 [AddTraceSource overloads](https://docs.microsoft.com/aspnet/core/api/microsoft.extensions.logging.tracesourcefactoryextensions) let you pass in a source switch and a trace listener.
 
 To use this provider, an application has to run on the .NET Framework (rather than .NET Core). The provider lets you route messages to a variety of [listeners](https://msdn.microsoft.com/library/4y5y10s7), such as the [TextWriterTraceListener](https://msdn.microsoft.com/library/system.diagnostics.textwritertracelistener) used in the sample application.
 
 The following example configures a `TraceSource` provider that logs `Warning` and higher messages to the console window.
 
-[!code-csharp[](logging/sample/src/TodoApi/Startup.cs?name=snippet_TraceSource&highlight=8-12)]
+[!code-csharp[](logging/sample//Startup.cs?name=snippet_TraceSource&highlight=8-12)]
```

Comment:
The comment in this snippet is tough to read because it's highlighted. I recommend starting the highlighting at line 9.